tests/e2e/specs/in-game/sevens.spec.js

```diff
@@ -1487,4 +1487,45 @@ describe('Playing sevens at the end of the deck', () => {
       scrap: [Card.SEVEN_OF_CLUBS],
     });
   });
+
+  it('Cannot play seven one-off if deck is empty', () => {
+    cy.setupGameAsP1();
+    cy.loadGameFixture(1, {
+      p0Hand: [],
+      p0Points: [],
+      p0FaceCards: [],
+      p1Hand: [Card.SEVEN_OF_CLUBS, Card.ACE_OF_CLUBS],
+      p1Points: [],
+      p1FaceCards: [],
+      topCard: Card.SIX_OF_HEARTS,
+      secondCard: Card.SEVEN_OF_HEARTS,
+      deck: [Card.FIVE_OF_DIAMONDS],
+    });
+    cy.drawCardOpponent();
+    cy.get('#deck').click();
+    cy.drawCardOpponent();
+    cy.get('[data-player-hand-card=7-0]').click();
+    cy.get('[data-move-choice=oneOff').should('have.class', 'v-card--disabled');
+  });
+
+  it("Cannot play last card of deck as 7 one-off when chaining 7's", () => {
```

Comment:
I know I suggested this name but it's nice to avoid the apostrophe so we can use single quotes
```suggestion
  it('Cannot play last card of deck as 7 one-off when chaining sevens', () => {
```